docs/intro/tutorial08.txt

```diff
@@ -22,10 +22,11 @@ Installing Django Debug Toolbar
 ===============================
 
 Django Debug Toolbar is a useful tool for debugging Django web applications.
-It's a third-party package maintained by the `Jazzband
-<https://jazzband.co>`_ organization. The toolbar helps you understand how your
-application functions and to identify problems. It does so by providing panels
-that provide debug information about the current request and response.
+It's a third-party package that has been maintained by the community
+organization, `Jazzband <https://jazzband.co>`_ and now `Django Commons
+<https://github.com/django-commons>`_. The toolbar helps you understand how
+your application functions and to identify problems. It does so by providing
+panels that provide debug information about the current request and response.
```

Comment:
```suggestion
It's a third-party package that is maintained by the community organization
`Django Commons <https://github.com/django-commons>`_. The toolbar helps you
understand how your application functions and to identify problems. It does so
by providing panels that provide debug information about the current request
and response.
```
Although I agree that Jazzband is still relevant, I don't think in the context of the tutorial of your first Django application having a formerly maintained by reference is useful
I think if we have more of a community package page in future it might make sense there